packages/rrdom/rollup.config.js

```diff
@@ -55,6 +55,7 @@ for (let config of baseConfigs) {
           name: config.name,
           format: 'iife',
           file: pkg.unpkg.replace(pkg.name, config.path),
+          extend: true,
```

Comment:
This is the only deliberate change I made: After changing the namespace, rollup threw an error that the package name is illegal unless I add this option. Tbh I'm not entirely sure what the problem here is but I don't think we need this particular build config anyway.